setup.cfg

```diff
@@ -0,0 +1,4 @@
+[qgis-plugin-ci]
+plugin_path = redistrict
+github_organization_slug = linz
+project_slug = qgis-plugin
```

Comment:
Shouldn't this be:

```suggestion
plugin_path = redistrict
github_organization_slug = north-road
project_slug = qgis-redistricting-plugin
```

Reply:
Sorry, my bad... Forgot the main repository is on north-road.